charts/argo/values.yaml

```diff
@@ -29,6 +29,7 @@ workflow:
     create: false  # Specifies whether a service account should be created
     annotations: {}
     name: "argo-workflow"  # Service account which is used to run workflows
+    pullSecrets: []  # Secrets with credentials to pull images from a private registry
```

Comment:
Should we just make this key `imagePullSecrets` too?

Reply:
@stefansedich are you suggesting renaming `pullSecrets` to `imagePullSecrets`, or are you suggesting just using the value that is defined for `images.pullSecrets`?